src/libraries/Native/Unix/System.Native/pal_process.c

```diff
@@ -191,6 +191,11 @@ static int SetGroups(uint32_t* userGroups, int32_t userGroupsLength, uint32_t* p
     return rv;
 }
 
+typedef union {
+    void (*handler)(int);
+    void (*action)(int, siginfo_t *, void *);
+} pal_sighandler;
+
 int32_t SystemNative_ForkAndExecProcess(const char* filename,
```

Comment:
this isn't actually used anywhere is it?

Reply:
The union or the method? The union is used few lines below and the method in `System.Native` library. I didn't want to make the union visible, so I kept it here instead of putting it in a header file.

Reply:
I think we can make the method a noop on browser

Reply:
I made it to not define HAVE_FORK on Browser, so that should make it just return `-1`.